ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java

```diff
@@ -141,6 +141,7 @@ public enum HiveOperationType {
   ALTERTABLE_UPDATECOLUMNS,
   ALTERTABLE_CREATEBRANCH,
   ALTERTABLE_CREATETAG,
+  ALTERTABLE_DROPBRANCH,
```

Comment:
nit can we pull this up, to keep it together with create branch

Reply:
Done.